webapp/src/app/app.component.ts

```diff
@@ -1,10 +1,61 @@
-import { Component } from '@angular/core';
+import { Component, NgZone } from '@angular/core';
+import { AngularFireAuth } from 'angularfire2/auth';
+import { AngularFireDatabase, FirebaseObjectObservable } from 'angularfire2/database';
+import { Observable } from 'rxjs/Observable';
+
+import { Profile } from './models/profile'
+
+// TODO: the following is temporary -- check with deast@
+import * as firebase from 'firebase/app';
+
+Profile
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
+  loginStatus: "loggedIn"|"loggedOut"|"loading" = "loading";
```

Comment:
actually it can't be an enum: https://github.com/angular/angular/issues/2885
but agree it should be a type!